lms/views/email.py

```diff
@@ -115,6 +123,13 @@ def set_preferences(self):
         email_preferences.sat = params.get("sat") == "on"
         email_preferences.sun = params.get("sun") == "on"
 
+        if email_preferences.mention_email_feature_enabled:
+            # Only set the mention_email_subscribed preference if the feature is enabled
+            # This avoids setting it to False if the feature is not yet enabled
+            email_preferences.mention_email_subscribed = (
+                params.get("mention_email_subscribed") == "on"
```

Comment:
@acelaya This is a proposal of how the client could send the value of the subscribe/unsubscribe.

Here is the same endpoint (form submission) that currently receives the "sat", "sun"... value for digest.

We could go for any other approach.

Reply:
I suppose a single form should work. I don't see a problem.